enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -622,18 +769,30 @@ customizing these parameters.
 For the time being, we may defer the ability to customize the parameters
 described above and address the question in a followup EP for tech preview.
 
-**Resolution**: TBD.
+**Resolution**: Deferred until tech preview.  For dev preview, we do not allow
+customization of the SMCP.  The Ingress Operator simply specifies the
+"gateway-controller" profile.  This profile enables Gateway API for ingress,
```

Comment:
I don' think we specify the "gateway-controller" profile for dev preview?

Reply:
Whoops, I forgot to update that.  Fixed in https://github.com/openshift/enhancements/compare/5ccb87fd14dd84eebfc38506dc1596a75eeccdfb..d60687328a0112a0472424afaf6ec4cc2611b98c.